sql/mysqld.cc

```diff
@@ -7006,9 +7010,22 @@ static int show_heartbeat_period(THD *thd, SHOW_VAR *var, char *buff,
   return 0;
 }
 
-
 #endif /* HAVE_REPLICATION */
 
+static int show_max_used_connections_time(THD *thd, SHOW_VAR *var, char *buff,
+                                 enum enum_var_type scope)
+{
+
+  char time_str[20];
+  get_date(time_str, GETDATE_DATE_TIME, max_used_connections_time);
+
+  var->type= SHOW_CHAR;
+  var->value= buff;
+
+  sprintf(buff, "%s", time_str);
+  return 0;
+}
+
```

Comment:
Functions should be diveded with 2 spaces(  we need to update that in Coding standard) @LinuxJedi